src/ngraph/builder/quantization/quantized_linear_matmul.hpp

```diff
@@ -37,6 +37,12 @@ namespace ngraph
 
             std::shared_ptr<Node> QuantizedLinearMatmulInteger(const std::shared_ptr<Node>& input0,
                                                                const std::shared_ptr<Node>& input1);
+
+            std::shared_ptr<Node>
```

Comment:
I still prefer a single builder with all inputs.

Reply:
With the current solution I can avoid creating `Constant` nodes in the graph if ONNX doesn't provide these optional values.